### DISCUSSION TITLE
Workflow task Send PDF

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16743] [General]
Been testing a bit workflows and it appears that Send PDF task doesnt deliver PDF even this PDF is visible in  Mails to Send panel.
With one exception, when I click Manual Sending this attachment goes correctly, but when I leave this email for cron, the PDF template is not attached.
The only error I can trace is:
PHP Warning:  bcadd(): bcmath function argument is not well-formed in /var/www/html/vendor/yetiforce/yetiforcepdf/lib/Math.php on line 68

Core file mentioned in log was not touched.
Can anyone confirm this please, so I can report bug or dig more?
It is on latest stable

Best

**Comment by ScottAndaman** [comment]
Exactly same situation here

S.

**Comment by garryatbrighton** [comment]
In my instance also not working as expected. Have not been using this sending method as it feels inconsistent. But checked s2b-git scenario and yes it not works. Also YF6.4

**Comment by krseguros** [comment]
The same happens to me,
I try to send a pdf but the mail arrives only without the pdf attached
![image](https://user-images.githubusercontent.com/96985476/230181122-08661614-6458-45bc-ba09-83a551e7dc9b.png)
![image](https://user-images.githubusercontent.com/96985476/230181226-f894c851-30e1-44d7-a002-ed8138ba2afe.png)
![image](https://user-images.githubusercontent.com/96985476/230181290-da4ebfb5-ba00-49a8-94ce-4a092f8e1b2e.png)
![image](https://user-images.githubusercontent.com/96985476/230181572-fdd80269-aed0-4b62-84f5-80701b17c041.png)

What would be the solution to this?

**Comment by apcloic** [comment]
Issue also confirmed on my side.
I've noticed that if you attach a PDF file to your email template, it is delivered but it does not offer the same functionnalities as send PDF via workflow.

**Comment by garryatbrighton** [comment]
migrated 6.4 to 6.5 and hit the same. Anyone resolved in the meantime?

**Comment by rskrzypczak** [comment]
In YF version 7.x I don't see this issue occuring.
